OpenRA.Mods.Common/Traits/Render/RenderSprites.cs

```diff
@@ -36,7 +36,9 @@ public class RenderSpritesInfo : IRenderActorPreviewInfo, ITraitInfo
 		[PaletteReference] public readonly string Palette = null;
 
 		[Desc("Custom PlayerColorPalette: BaseName")]
-		public readonly string PlayerPalette = "player";
+		[PaletteReference("IsPlayerPalette")] public readonly string PlayerPalette = "player";
```

Comment:
Yeah, I said it should be "true" for a reason.

We should probably merge this with the `Palette` property above, but that can be done in a follow-up PR.